package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neutron-org/neutronjsplus",
-  "version": "0.0.16",
+  "version": "0.1.0-rc4",
```

Comment:
I think we should have a mapping neutron version <-> neutronjsplus version somewhere, for example in the repo readme. let's add it? at least starting with this version: neutron v2.0.0 <-> neutronjsplus 0.1.0